packages/volto/src/components/manage/Form/Form.jsx

```diff
@@ -982,6 +1002,8 @@ class Form extends Component {
                   )}
                   {map(schema.fieldsets[0].fields, (field) => (
                     <Field
+                      widgets={this.props.widgets}
+                      component={this.props.component}
```

Comment:
Why do you pass `component` to the Field here and on line 949? I don't see where it is used?

Reply:
This can be left empty. But if used this is a way to provide an inner (dumb) widget to your component.